src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityMultiblockPart.java

```diff
@@ -70,6 +71,12 @@ public MultiblockControllerBase getController() {
 
     public ICubeRenderer getBaseTexture() {
         MultiblockControllerBase controller = getController();
+        if (controller != null) {
+            this.hatchTexture = controller.getBaseTexture(this);
+        }
+        if (controller == null && this.hatchTexture != null){
+            return this.hatchTexture;
+        }
```

Comment:
pontential problems: if all controllers have been removed. hatchTexture will be still rendered if not null